source/tutorials/metadata.rst

```diff
@@ -0,0 +1,138 @@
+Metadata in QIIME 2
+===================
+
+Metadata enables users of QIIME 2 to supplement their analyses with information relevant to their study. This manifests itself in many ways: indicating the barcode sequence when demultiplexing sequences, sorting and grouping taxonomic assignments in the taxonomic bar plots, or exploring diversity metrics with visualizations like ``alpha-group-significance`` or ``beta-group-significance``. In QIIME 1, metadata (also known as the metadata mapping file), was a user-defined file that contained these study-specific fields. QIIME 2 expands upon this idea, allowing users to provide their own study metadata (via a TSV file), or, by viewing other study-specific artifacts as metadata. Examples of both are presented in the following.
```

Comment:
I wouldn't really call metadata a supplement - it's essential for getting any insight from the data. 

I wonder if this first paragraph should be something like: *Metadata provides the key to gaining biological insight from your raw data. In QIIME 2, sample metadata includes technical details, such as the DNA barcodes that were used for each sample, or descriptions of the samples, such as which subject, time point and body site each sample came from in a human microbiome time series. Feature metadata is often a feature annotation, such as the taxonomy assigned to a sequence variant. Sample and feature metadata are used by many methods and visualizers in QIIME 2.*

Then, beginning with *In QIIME 1*, should that be in one of our "QIIME 1 users" directives? 

*metadata mapping file* -> *mapping file, or sample metadata mapping file*